### DISCUSSION TITLE
Public Airtable Base

### DISCUSSION THREAD

**Discussion by ghost** [cmdalbem/escapista#5] [Q&A] [answered]
Can i duplicate your Airtable base for local development?

**Comment by cmdalbem** [comment, marked as answer]
Yes sure, go ahead! Do you have access to it?
Sorry I hadn't noticed that the code wouldn't be executable without our Airtable database template.

**Reply by ghost** [reply]
I don't have access, can you create an empty template maybe?

**Reply by cmdalbem** [reply]
Sure, [here it is](https://airtable.com/invite/l?inviteId=invqBwuoFbT8ZV2Gv&inviteToken=e23566c0e747fc7dc63f780476664e475a65be30fc3d71f2232b69577e42fa41).
I've removed all our data and simplified the structures to make it easier to understand. I've tested it here locally and it works well, but you'll need this latest commit https://github.com/cmdalbem/escapista/commit/ffa10f8b6768ed80bc0697b465a930ca25094e21  where I remove so hardcoded references for some of our channels.

Let me know if it works for you, so then I can add this to the project documentation :)

**Reply by ghost** [reply]
Wow, thanks time to contribute :smile_cat:

**Reply by cmdalbem** [reply]
I have to admit I didn't think anyone would be interested in the code so fast haha I'm glad there's
I'll move here to Github some documentation I have outside, mainly known bugs if you happen to encounter them, and some ideas too
